### PR TITLE
add lightgallery v1.1.5 with git autoupdate

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Can you update commit message to some kind of this:
`add lightgallery v1.1.5 with git autoupdate, close #5016`

**Comment by maruilian11** [comment]
ok

**Comment by PeterDaveHello** [comment]
Should add all the files under `light-gallery` in this case.

**Comment by maruilian11** [comment]
ok

**Comment by PeterDaveHello** [comment]
Why didn't you update the real files?

**Comment by maruilian11** [comment]
plz take a look.

File: ajax/libs/lightgallery/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      "type": "MIT",
+      "url": "http://opensource.org/licenses/MIT"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sachinchoolur/lightGallery.git",
+    "basePath": "light-gallery",
+    "files": [
+      "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/lightgallery/package.json]: Should not use `**/*` in non-dist folder.

**Comment by maruilian11** [comment]
is that ok?

**Comment by PeterDaveHello** [comment]
Would you please also add this field in ?

```js
reruiredFiles: [
  "css/lightGallery.min.css"
]
```

Because they didn't provide the minified file, this can remind our maintainer to minify it in future.

**Comment by maruilian11** [comment]
ok

**Pull request merged**